r-package/grf/tests/testthat/test_causal_forest_tuning.R

```diff
@@ -38,3 +38,28 @@ test_that("causal forest tuning only cross-validates null parameters", {
     expect_equal(as.numeric(tunable.params["min.node.size"]), min.node.size)
     expect_equal(as.numeric(tunable.params["imbalance.penalty"]), imbalance.penalty)
 })
+
+test_that("local linear causal forest tuning returns lambda and decreases error", {
```

Comment:
It would also be nice to add a test analogous to "output of tune local linear forest is consistent with prediction output" (in `test_local_linear_forest`). This test ended up helping us catch an important bug.